docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -1,22 +1,30 @@
 {
```

Comment:
<p>Maybe worth adding a markdown cell above briefly stating that users should either store their API token in an environment variable or run the cell below replacing their "key" with the token retrieved from <a href="https://superstaq.super.tech/" target="_blank">https://superstaq.super.tech</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done!</p>